dartagnan/src/main/java/com/dat3m/dartagnan/program/event/lang/linux/LKMMLock.java

```diff
@@ -10,7 +11,7 @@ public class LKMMLock extends SingleAccessMemoryEvent {
     public LKMMLock(Expression lock) {
         // This event will be compiled to LKMMLockRead + LKMMLockWrite
         // and each of those will be assigned a proper memory ordering
-        super(lock, "");
+        super(lock, TypeFactory.getInstance().getIntegerType(32), "");
```

Comment:
Are you sure spinlocks in the kernel are 32 bits?

Reply:
Didn't we have a discussion about this? Anyway, we mock the locks with our own header and that uses `int` internally. I also think that other sizes caused problems in some tests (not sure though)